CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Protagonist Changelog
 
+## 2.0.0-pre.4
+
+### Enchancements
```

Comment:
```suggestion
### Enhancements
```

Reply:
I can't commit this since you are the owner.